### ISSUE TITLE
The evaluation method of FedAVG

### ISSUE THREAD

**Issue by FibonacciZ** [TsingZ0/PFL-Non-IID#60]
In Fedavg paper, Fedavg should evaluate the global model. In your code, Why FedAVG evaluate client models.

**Comment by countingeyes** [comment]
Because the global model has been sent to all clients.
![image](https://user-images.githubusercontent.com/75488849/222938258-c1c66309-97b1-4ab7-80cf-97385c7ee88a.png)

**Comment by countingeyes** [comment]
Hi FibonacciZ, did you test FedAvg and FedBABU with the non_iid dataset? I met some problems with the test acc.  The test acc in my experiment first rises and then falls. Is it overfitting? Besides the test acc drops sharply and randomly. Is the learning_rate set too big?
![image](https://user-images.githubusercontent.com/75488849/222938732-4c3d1088-d95d-4ab1-b82a-08c65bfd3e06.png)
![image](https://user-images.githubusercontent.com/75488849/222938782-114133b5-6415-4016-960d-a6aa62ca06c9.png)
The hyperparameters I used is resnet18  cifar10  100clients 0.1join_train_rate 100batchsize 0.1lr 0.998lr_decay 0.9momenton 5e-4weght_decay~
Looking forward to your reply!

**Comment by FibonacciZ** [comment]
Thanks for your answer. In my opinion,  the large fluctuations in 0~400 rounds were caused by large learning_rate. My learning_rate is 0.005, never found the results like yours.

**Comment by countingeyes** [comment]
> @countingeyes . Thanks for your answer. In my opinion, the large fluctuations in 0~400 rounds were caused by large learning_rate. My learning_rate is 0.005, never found the results like yours.

Thanks for your advice! I tried the smaller learning_rate like 0.005, 0.01, and 0.05 but found that the acc curve was smoother but lower. The best acc are {0.45, 0.75, 0.82} with the learning rate {0.005, 0.05, 0.1}.  It seems that the too small learning rate will fail to converge to the optimal solution. Will your experiments be like this？ Or if you need better results, maybe you could try a bigger lr~

**Comment by TsingZ0** [comment]
#58

**Comment by Wangyue2333** [comment]
@countingeyes May I ask how your Leraning_rate and lr_decay are ultimately set
